app/controllers/diseases_controller.rb

```diff
@@ -12,7 +12,11 @@ def show
     @disease = Disease.where("lower(name) LIKE?", "%#{permitted_params[:disease].downcase}%").first
   end
 
+  def show_attr
+    @disease_attr = Disease.where("lower(name) LIKE?", "%#{permitted_params[:disease].downcase}%").select("id, #{permitted_params[:attribute].downcase}").first
```

Comment:
I think we can look at limiting `attribute` to strictly only available columns, so as to prevent a NoColumn error or something.

Reply:
Sure, a helper with this validation will do the job.